var/spack/repos/builtin/packages/mvapich2/package.py

```diff
@@ -83,7 +83,15 @@ class Mvapich2(AutotoolsPackage):
 
     variant(
         'fabrics',
-        description='The fabric enabled for this build',
+        description='Select the fabric to be enabled for this build.'
+        'Please note that if you have verbs (either from OFED or MOFED), '
+        'PSM or PSM2 installed on the system already, you may need to '
+        'setup external packages in the package.yaml file for rdma-core '
+        'psm or opa-psm2, depending on your chosen fabrics. This is '
+        'recommended to avoid unexpected runtime failures due to conflicts '
+        'between the version of the fabric drivers spack installs and the '
+        'fabric version available on the system. See the homepage url above '
+        'for more info.',
```

Comment:
The description should be short, given the contexts in which it is shown. Can you remove references like "above" or turn this into a comment?

Reply:
@adamjstewart Should we perhaps allow for an "extra comment" field to be shown in `spack info`? The changes here are good but, given how a very long description is currently layout on the screen, they don't look very nice.

Reply:
Nah, I don't think we need an extra comment field